include/at_cmd_parser.h

```diff
@@ -0,0 +1,54 @@
+/**@file at_cmd_parser.h
+ *
+ * @brief Basic parser for AT commands.
+ *
+ */
+
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#ifndef AT_CMD_PARSER_H_
+#define AT_CMD_PARSER_H_
+
+#include <zephyr.h>
```

Comment:
Why do you include this header? Could you include `zephyr/types.h` instead?

Reply:
Fixed